package.json

```diff
@@ -140,6 +152,35 @@
         "messages/**/*.md"
       ],
       "output": []
+    },
+    "test:command-reference": {
+      "command": "node --loader ts-node/esm --no-warnings=ExperimentalWarning \"./bin/dev.js\" commandreference:generate --erroronwarnings",
+      "files": [
+        "src/**/*.ts",
+        "messages/**",
+        "package.json"
+      ],
+      "output": [
+        "tmp/root"
+      ]
+    },
```

Comment:
this can be removed, we haven't decided how to doc the MCP server yet.

Reply:
dev-scripts requires it to be there. I think because it sees the oclif configuration and assumes it's just another plugin